core/src/test/scala/com/stripe/dagon/RealNumberTest.scala

```diff
@@ -29,8 +29,14 @@ object RealNumbers {
     def +[A1 >: A: Ordering](a: A1): SortedList[A1] =
       new SortedList((a :: toList).sorted)
 
-    def -[A1 >: A](a: A1): SortedList[A1] =
-      filterNot(_ == a)
+    def removeFirst[A1 >: A](a: A1): SortedList[A1] =
```

Comment:
this was a bug: SortedList is a multiset, not a set. You can't use filter to remove 1 item. @avibryant